app.js

```diff
@@ -1,9 +1,14 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controller");
-const { invalidRequest } = require("./controllers/errorHandle.controller");
+const {
+  invalidRequest,
+  customError,
+  BadRequest,
+} = require("./controllers/errorHandle.controller");
 const { getArticleById } = require("./controllers/articles.controller");
 
 const app = express();
+app.use(express.json());
 
 app.get("/api", (request, response, next) => {
```

Comment:
This does not need to exist at this stage, its nothing something that was asked for in the ticket and you will need this endpoint in a later ticket.